docs/reference-architectures/virtual-machines-linux/multi-vm.md

```diff
@@ -3,104 +3,93 @@ title: Run load-balanced VMs on Azure for scalability and availability
 
-The architecture has the following components: 
+The architecture has the following components:
 
-* **Availability set**. The [availability set][availability set] contains the VMs. This makes the VMs eligible for the [availability service level agreement (SLA) for Azure VMs][vm-sla]. For the SLA to apply, you need a minimum of two VMs in the same availability set. 
+* **Resource group.** A [*resource group*][resource-manager-overview] is a container that holds related resources. You usually create resource groups for different resources in a solution based on their lifetime, and who will manage the resources.
 * **Virtual network (VNet) and subnet.** Every VM in Azure is deployed into a VNet that is further divided into subnets.
 * **Azure Load Balancer**. The [load balancer] distributes incoming Internet requests to the VM instances. The load balancer includes some related resources:
   * **Public IP address**. A public IP address is needed for the load balancer to receive Internet traffic.
   * **Front-end configuration**. Associates the public IP address with the load balancer.
   * **Back-end address pool**. Contains the network interfaces (NICs) for the VMs that will receive the incoming traffic.
-* **Load balancer rules**. Used to distribute network traffic among all the VMs in the back-end address pool. 
-* **Network address translation (NAT) rules**. Used to route traffic to a specific VM. For example, to enable remote desktop protocol (RDP) to the VMs, create a separate NAT rule for each VM. 
-* **Network interfaces (NICs)**. Each VM has a NIC to connect to the network.
+* **Load balancer rules**. Used to distribute network traffic among all the VMs in the back-end address pool.
+* **VM Scaleset**. A [VM scaleset][vm-scaleset] contains a set of VMs with identical settings, used to host the same workload. Scaelsets allow to scale the number of VMs out and in manually, or automatically based on predefined rules.
+* **Availability set**. The [availability set][availability set] contains the VMs. This makes the VMs eligible for the [availability service level agreement (SLA) for Azure VMs][vm-sla]. For the SLA to apply, you need a minimum of two VMs in the same availability set.
 * **Storage**. If you are not using [managed disks](/azure/storage/storage-managed-disks-overview), storage accounts hold the VM images and other file-related resources, such as VM diagnostic data captured by Azure.
 
 You can download a [Visio file](https://aka.ms/arch-diagrams) of this architecture.
 
 > [!NOTE]
 > Azure has two different deployment models: [Resource Manager][resource-manager-overview] and classic. This article uses Resource Manager, which Microsoft recommends for new deployments.
->
 
 ## Recommendations
 
 Your requirements might differ from the architecture described here. Use these recommendations as a starting point. 
 
-
 ### Availability set recommendations
 
```

Comment:
Change this section to "Scale set recommendations"

Instead of "create at least 2 VMs" say something like "scale out to at least 2 VMs" ?

Mention availability sets (w/out VMSS) somewhere in the as an alternative to using VMSS? If we think there is a use case - e.g., SQL Server where you don't want autoscale